docs/user-guide/environment-variables.md

```diff
@@ -80,7 +80,7 @@ These environment variables may or may not be available depending on what plugin
 
 | Name | Description |
 |------|-------|
-| SD_ARTIFACTS_DIR | Location of built/generated files |
+| SD_ARTIFACTS_DIR | Location of built/generated files. _Note: The UI will only display artifacts when builds complete successfully. If a build fails, you can still access the artifacts, but it will need to be done through the store API (e.g.: `https://store.screwdriver.cd/v1/builds/12345/ARTIFACTS/SD_ARTIFACT.zip`). You will need to pass in a token along with the request. See the [API docs](./api.md) for more details._ |
```

Comment:
this goes together with `SD_ZIP_ARTIFACTS` https://github.com/screwdriver-cd/guide/pull/336/commits/2bb2e07984fe7b19fc546a93a27853ff9367588c#diff-6d6e8ea6c3545b2ffca783257d7e4240R61

Reply:
I think this is true whether or not you chose to zip artifacts
so technically needs to be in both places

Reply:
this file won't exist unless `SD_ZIP_ARTIFACTS` is turned on.

Reply:
it's just an example file name
can also have `environment.json`, etc
https://cd.screwdriver.cd/pipelines/1/builds/152628/steps/sd-teardown-screwdriver-artifact-bookend

Reply:
updated to have a more generic file name

Reply:
`The UI will only display artifacts when builds complete successfully` is not entirely correct. When build is not in completed state UI does not display artifacts. Also another case is aborted builds.